_unittest/test_41_3dlayout_modeler.py

```diff
@@ -652,6 +654,7 @@ def test_41_test_create_polygon(self):
 
     @pytest.mark.skipif(not config["use_grpc"], reason="Not running in COM mode")
     @pytest.mark.skipif(config["desktopVersion"] < "2023.2", reason="Working only from 2023 R2")
+    @pytest.mark.skipif(is_linux, reason="PyEDB failing in Linux")
```

Comment:
```suggestion
    @pytest.mark.skipif(is_linux, reason="PyEDB is failing in Linux.")
```
Is failing or has failed?